khard/carddav_object.py

```diff
@@ -57,6 +57,13 @@ def convert_to_vcard(name, value, allowed_object_type):
                      " must be a string or a list with strings.")
 
 
+def multi_property_key(item):
```

Comment:
a docstring for this one please

Reply:
While writing the docstring here (proving its usefulness) I realised a couple of things. Chiefly, I'm currently returning the label to sort here, but I'm not sure whether that's sensible.

Returning the value would make more sense. Better still I think would be sorting by type so that labelled are either before or after unlabelled, then by label for labelled and content for unlabelled. I have an idea for that.

Reply:
returning the value sounds good

Reply:
I managed to make it sort unlabelled objects first (by value), then the labelled ones by label. Would you prefer sorting all of them by value? It's an easy change.

Reply:
I don't really care and it is easy to change later.